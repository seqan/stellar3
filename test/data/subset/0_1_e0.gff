chr2	Stellar	eps-matches	202	253	100	+	.	1;seq2Range=498,549;eValue=7.02467e-22;cigar=52M;mutations=
chr1	Stellar	eps-matches	266	316	100	+	.	1;seq2Range=251,301;eValue=2.00885e-21;cigar=51M;mutations=
chr1	Stellar	eps-matches	184	233	100	+	.	1;seq2Range=346,395;eValue=6.60326e-21;cigar=50M;mutations=
chr2	Stellar	eps-matches	169	218	100	+	.	1;seq2Range=438,487;eValue=7.59009e-21;cigar=50M;mutations=
chr2	Stellar	eps-matches	370	420	100	-	.	1;seq2Range=1101,1151;eValue=2.30907e-21;cigar=51M;mutations=
chr1	Stellar	eps-matches	106	155	100	-	.	1;seq2Range=1411,1460;eValue=6.60326e-21;cigar=50M;mutations=
chr2	Stellar	eps-matches	517	566	100	-	.	1;seq2Range=1896,1945;eValue=7.59009e-21;cigar=50M;mutations=
chr2	Stellar	eps-matches	149	198	100	-	.	1;seq2Range=1193,1242;eValue=7.59009e-21;cigar=50M;mutations=
