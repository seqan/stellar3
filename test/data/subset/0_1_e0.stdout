I/O options:
  database file   : multi_seq_ref.fasta
  query file      : query_e0.fasta
  alphabet        : dna5
  output file     : subset/0_1_e0.gff
  output format   : gff

User specified parameters:
  minimal match length             : 50
  maximal error rate (epsilon)     : 0 = (0/1)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500

Calculated parameters:
  k-mer length : 32
  s^min        : 50
  threshold    : 19
  distance cut : 50
  delta        : 16
  overlap      : 0

Loaded 1 query sequence.
Loaded sequence chr1.
Loaded sequence chr2.

All matches resulting from your search have an E-value of: 
        1.71539e-20 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  chr1
    # SWIFT hits      : 2
    Longest hit       : 51
    Avg hit length    : 50
  chr2
    # SWIFT hits      : 2
    Longest hit       : 52
    Avg hit length    : 51
  chr1, complement
    # SWIFT hits      : 1
    Longest hit       : 50
    Avg hit length    : 50
  chr2, complement
    # SWIFT hits      : 3
    Longest hit       : 51
    Avg hit length    : 50

# Eps-matches     : 8
Longest eps-match : 52
Avg match length  : 50
